CHANGELOG

```diff
@@ -1,3 +1,13 @@
+21 January 2015 - Release 0.14.4
+===================================
+
+Highlights:
+
+* Adds removeAll:  removeAllFoundIn: and removeAllSuchThat: to the collection API
+* Adds one test for each of those methods.
```

Comment:
This neither. CHANGELOG only contains general description of what the release contains.
API is is in API-CHANGES.txt.